### ISSUE TITLE
native search does not match hidden rows

### ISSUE THREAD

**Issue by debrice** [facebook/fixed-data-table#224]
When using native <kbd>Ctrl</kbd>+<kbd>f</kbd> web browser search functionality does not find matches in the table rows while scrolling reveals matching rows, it's counter-intuitive.

**Comment by rmilesson** [comment]
Yeah, I think you'll have to go poke whichever vendor of your browser of choice for that since it's their search functionality.
